frontend/src/components/course-search/CourseRow.tsx

```diff
@@ -36,10 +37,51 @@ export default memo(function CourseRow(props: {
         props.updateDetailsSize(detailsBounds.height);
     }, [detailsBounds?.height, props.updateDetailsSize, props.expand]);
 
+    if (props.fromOtherTerm) {
+        return (
+            <div className={Css.padder}>
+                <div
+                    className={classNames(Css.box, {
+                        [Css.expand!]: props.expand,
```

Comment:
you don't need the non-null assertion here

Reply:
I will fix it.